Dockerfile

```
@@ -0,0 +1,24 @@
+FROM python:buster as base
+RUN apt-get update
+RUN apt-get install -y curl
+ENV WEBAPP_FOLDER=/opt/todoapp
+RUN mkdir $WEBAPP_FOLDER
+WORKDIR $WEBAPP_FOLDER
```

Minor, but worth noting that the mkdir will happen automatically if the folder doesn't exist with `WORKDIR`
```suggestion
WORKDIR $WEBAPP_FOLDER
```